valhalla/app/controllers/valhalla/downloads_controller.rb

```diff
@@ -51,7 +51,7 @@ def prepare_file_headers
     response.headers['Content-Type'] = file_desc.mime_type.first.to_s
     response.headers['Content-Length'] ||= binary_file.size.to_s
     # Prevent Rack::ETag from calculating a digest over body
-    response.headers['Last-Modified'] = modified_date
+    response.headers['Last-Modified'] = modified_date if modified_date
```

Comment:
It's super weird this is blank, I wonder if there's a bug.

Reply:
Agreed, I just created #344 to figure out why it's blank.